package.json

```diff
@@ -39,6 +39,6 @@
     "typescript": "^3.6.2"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
I forgot to remove Node 8 from Travis matrix. I will fix it if this change is accepted.